test/validators.js

```diff
@@ -114,7 +114,7 @@ describe('Validators', () => {
         'multiple..dots@gmail.com',
         'wrong()[]",:;<>@@gmail.com',
         '"wrong()[]",:;<>@@gmail.com',
-        'username@domain.com�',
+        'username@domain.com ',
```

Comment:
This seems like an irrelevant change made by your IDE automatically:
```suggestion
        'username@domain.com�',
```